docs/src/main/asciidoc/intro.adoc

```diff
@@ -429,6 +429,18 @@ To do this you can use respectively `ZipkinAutoConfiguration.REPORTER_BEAN_NAME`
 include::../../../spring-cloud-sleuth-zipkin/src/test/java/org/springframework/cloud/sleuth/zipkin2/ZipkinAutoConfigurationTests.java[tags=override_default_beans,indent=0]
 ----
 
+The default web sender uses a specific `RestTemplate` that you can customize by providing a bean of type
+`ZipkinRestTemplateCustomizer`. If you wish to use a more specific RestTemplate (e.g. a OAUth2RestTemplate) 
+then you need to give your bean the specific name 
+`ZipkinRestTemplateSenderConfiguration.REST_TEMPLATE_BEAN_NAME`. 
```

Comment:
Update this when you move the property to the other class

Reply:
done